### PR TITLE
New sprites

### PR DESCRIPTION
**Pull request by RuthAnn1**
I added the new sprites with the backgrounds changed and made sure the rock_sprites path reflected the new background.  I also changed the black color on the config file to match.  I am not sure what other files need to be changed.  @dinaolmelak  please take a look and let me know.  Thank you. 

Lastly,  I added 3 more sound effects.  I made a sound that can loop when the players are playing the game.  A sound effect for the choose pet screen and and sound effect for when the pet is mad.  They are all in mp3 format and good to go. :)

### CODE REVIEW

File: game_util/pet_config.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ class PetConfig:
     FONT = '../my_pet/theme_items/Starborn.ttf'
 
     # Colors
-    BLACK = (0, 0, 0)
+    BLACK = (0, 0, 5)
```
Comment (dinaolmelak) [inline game_util/pet_config.py:16]: I would say give this a different Name like `SPRITE_BG` 🤔 cause that way we would not confuse that with actual black. And then revert black back to `(0, 0, 0)`

File: sprites/rock_sprites_old.png
Comment (dinaolmelak) [inline sprites/rock_sprites_old.png:1]: Did you rename this? 🤔

**Review by dinaolmelak — CHANGES_REQUESTED** [review]
Just the black in pet config should change.

File: game_util/pet_config.py (hunk 1 above)
Comment (RuthAnn1) [reply game_util/pet_config.py:16]: Sounds good

File: sprites/rock_sprites_old.png
Comment (RuthAnn1) [reply sprites/rock_sprites_old.png:1]: yes because that is the old one and we want the new one to keep the same path. Correct?

Comment (dinaolmelak) [reply sprites/rock_sprites_old.png:1]: Yes 😄

**Review by dinaolmelak — APPROVED** [review]
LGTM

**Pull request merged**